VideoAPI/VideoApi.UnitTests/Controllers/Message/MessageControllerTestBase.cs

```diff
@@ -0,0 +1,35 @@
+using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+using Testing.Common.Helper.Builders.Domain;
+using Video.API.Controllers;
+using VideoApi.DAL.Commands.Core;
+using VideoApi.DAL.Queries.Core;
+using Task = System.Threading.Tasks.Task;
+
+
+namespace VideoApi.UnitTests.Controllers
+{
+    public class MessageControllerTestBase
+    {
+        protected Mock<IQueryHandler> queryHandler;
+        protected Mock<ICommandHandler> commandHandler;
+        protected Mock<ILogger<MessageController>> logger;
+        protected MessageController messageController;
+        protected VideoApi.Domain.Conference TestConference;
+
+        [SetUp]
+        public void TestInitialize()
```

Comment:
There is no need to have unit tests for controllers and mock everything. This endpoint is not complex so we do not need unit tests with mocks here

Reply:
Removed the test